framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/template/OpenShiftTemplatePropertiesLoader.java

```diff
@@ -0,0 +1,52 @@
+package org.kie.cloud.openshift.template;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+class OpenShiftTemplatePropertiesLoader {
```

Comment:
Not sure what it is about this class, but I don't like it. Perhaps it's the fact that all methods are `static`, yet it is not a singleton? Perhaps it's the way how `static load()` relies on side-effects of `static loadTemplate...()`?

The entire thing could essentially just be a lazy-loaded field.

Reply:
> The entire thing could essentially just be a lazy-loaded field.

Not sure what you mean, but I did some simplification and made the constructor private..